extension.json

```diff
@@ -15,6 +15,9 @@
 	"AutoloadNamespaces": {
 		"BootstrapComponents\\": "src/"
 	},
+	"TestAutoloadNamespaces": {
+		"BootstrapComponents\\Tests\\": "tests/phpunit/"
+	},
```

Comment:
This is necessary for unit tests to load classes, otherwise they fail for MW 1.41+.